cs/frontend.php

```diff
@@ -0,0 +1,154 @@
+    'noSpectators' => 'Nikdo se zrovna nedívá.',
+    'replayAvailable' => 'Pusťte si záznam na',
+    'oldVer' => 'Máte starou verzi, pomocí CTRL+F5 si načtěte novou!',
+    'oldVer2' => 'Máte starou verzi! Načtěte aplikaci znovu pomocí {key}',
+    'privateRoom' => 'Tohle je vaše soukromá místnost, kde můžete nerušeně hrát. Veřejné místnosti najdete v Lobby.',
+    'restartInfo' =>  'Začněte znovu pomocí F4, nebo si nastavte vlastní klávesu.',
+    'joinLinkInfo' => 'Tahle místnost je soukromá. Ostatní se můžou připojit jen pomocí tohoto odkazu:',
+
+    //Dynamic elements
+    'received' => 'Dostali jste', //max 15
+    'finesse' => 'Obratnost',   //max 15
+
+    //List of rooms
+    'roomName' => 'Název místnosti',
+    'games' => 'Her', //number of games played in a room
+    'players' => 'Hráčů', //number of players
+
+    //Results
+    'name' => 'Jméno',
+    'wins' => 'Výher',
+    'time' => 'Čas', //as game duration
+    'received' => 'Dostal/a',
+    'sent' => 'Odeslal/a',
+    'blocks' => 'bloků', //as number of blocks placed
+    'ren' => 'REN',
+
+    //Practice mode result
+    'gameTime' => 'čas', //in Sprint time: xx.xx
+    'see' => 'Zobrazit', //in 'See 40L leaderboard'
+    'leaderboard' => 'žebříček',
+
+    //Less common strings, can stay untranslated
+    'warning2' => 'VAROVÁNÍ',
+    'connected' => 'Připojeno',
+    'client' => 'klient',
+    'notConnected' => 'NEPŘIPOJENO',
+    'connLost' => 'Spojení ztraceno!',
+    'lastGame' => 'Poslední hra:',
+    'roomSettings' => 'Nastavení místnosti:',
+    'attack' => 'Útok',
+    'combo' => 'Kombo',
+    'solid' => 'Plné',
+    'clear' => 'Průhledné',
+    'mode' => 'Režim',
+    'garbage' => 'Garbage',
+    'garbageDelay' => 'ProdlevaGarbage',
+    'messiness' => 'Nepořádnost',
+    'replay' => 'Přehrát',
+    'rep' => 'Rep', //max 3, short for Replay
+    'rec' => 'Rec.', //max 4, short for Received
+    'enterNullDAS' => 'Zadejte hodnotu DAS z NullpoMina:',
+    'suggestedIs' => 'Doporučuje se DAS',
+    'applyConfirm' => 'Použít?',
+    'invalidDAS' => 'Neplatná hodnota DAS, návrat k původní hodnotě!',
+    'settingsChanged' => 'Změnili jste nastavení během hry, záznam je neplatný.',
+    'segment' => 'Segment',
+    'duration' => 'Trvání',
+    'roomFull' => 'Zdá se, že místnost je plná',
+    'lobbyInfo' => 'Tahle místnost je sdílená s {discord}.',
+    'newHost' => 'Teď jste správce této místnosti.',
+    'badRoom' => 'Tahle místnost už neexistuje, připojování k výchozí místnosti!',
+    'stngsChanged' => 'Nastavení změněno správcem',
+    'stngsCustom' => 'Vlastní nastavení',
+    'aSpec' => 'Už se díváte!',
+    'aPlay' => 'Už hrajete!',
+    'repFail' => 'Nahrávku se nepodařilo uložit.',
+    'repInChat' => 'Nahrávku jste poslali do chatu. Chcete-li si uchovat, zkopírujte text v rámečku a vložte ho do nějakého textového editoru.',
+    'repTxtInfo' => 'Zkopírovaný soubor si poté můžete pustit zde v přehrávači.'
```

Comment:
Accept this to fix the syntax error (missing `,`)
```suggestion
    'repTxtInfo' => 'Zkopírovaný soubor si poté můžete pustit zde v přehrávači.',
```